src/main/resources/application.yaml

```diff
@@ -2,4 +2,8 @@ spring:
   freemarker:
     template-loader-path: "classpath:/templates"
     suffix: ".ftl"
-
+  datasource:
+    url: jdbc:postgresql://localhost:7777/gmail
```

Comment:
use env variables for secrets, with default values for dev.
https://github.com/PNUdev/shpalta-if/blob/master/src/main/resources/application.yaml#L8